### PR TITLE
[WIP] Enhance speech translation for IWSLT2018

### PR DESCRIPTION
**Pull request by hirofumi0810**
- [x] Add BPE ST model
- [x] Add audio segmentation for development sets
- [x] Fix BLEU scoring
- [ ] Add other metrics (METEOR/TER)

### CODE REVIEW

File: egs/iwslt18st/st1/local/run_spm.sh [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+# rnnlm related
+lm_weight=0.3
+
+# decoding parameter
+beam_size=10
+penalty=0.0
+maxlenratio=0.0
+minlenratio=0.0
+ctc_weight=0
+recog_model=acc.best # set a model to be used for decoding: 'acc.best' or 'loss.best'
```
Comment (sw005320) [inline egs/iwslt18st/st1/local/run_spm.sh]: this should be model.xxx
please check diff with other similar script, andcheck it carefully
also, please perform some small test before PR.

File: egs/iwslt18st/st1/run.sh [hunk 2]
```diff
@@ -56,7 +56,7 @@ penalty=0.0
 maxlenratio=0.0
 minlenratio=0.0
 ctc_weight=0
-recog_model=model.acc.best # set a model to be used for decoding: 'model.acc.best' or 'model.loss.best'
+recog_model=acc.best # set a model to be used for decoding: 'acc.best' or 'loss.best'
```
Comment (sw005320) [inline egs/iwslt18st/st1/run.sh]: this should be model.xxx

File: egs/iwslt18st/st1/local/download_and_untar.sh [hunk 3]
```diff
@@ -39,117 +39,110 @@ if [ ! -d "$data" ]; then
...
   # sizes of the archive files in bytes.
   sizes="56896558179"
 
-  if [ -f $data/iwslt-corpus.zip ]; then
-    size=$(/bin/ls -l $data/iwslt-corpus.zip | awk '{print $5}')
-    size_ok=false
-    for s in $sizes; do if [ $s == $size ]; then size_ok=true; fi; done
+  if [ -f $data/$set/iwslt-corpus.zip ]; then
+    size=$(/bin/ls -l $data/$set/iwslt-corpus.zip | awk '{print $5}')
+    # size_ok=false
```
Comment (sw005320) [inline egs/iwslt18st/st1/local/download_and_untar.sh]: remove comment out lines

File: egs/iwslt18st/st1/local/run_spm.sh (hunk 1 above)
Comment (hirofumi0810) [reply egs/iwslt18st/st1/local/run_spm.sh]: Fixed it.

File: egs/iwslt18st/st1/run.sh (hunk 2 above)
Comment (hirofumi0810) [reply egs/iwslt18st/st1/run.sh]: Fixed it.

**Pull request closed without merging**